roles/kube_prometheus/files/prometheus_ingress.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: "kube-prometheus-prometheus-ingress"
```

Comment:
Could it  make sense to have the `/_/grafana/` definition in the very same file? We don't need 2 Ingress objects, right?